enterprise/utils.py

```diff
@@ -1446,10 +1445,7 @@ def get_platform_logo_url():
     Return an absolute URL of the platform logo using the branding api
     """
     # Return fake URL for tests rather than mock get_logo_url for every test using EnterpriseCustomer
-    return urljoin(
```

Comment:
I would just want to have what was the reason it was added in the first place? Just wanted to make sure we are not missing an edge case.

Reply:
I believe this was done in this way because `get_logo_url` returned a relative path to the platform logo URL at time of implementation and the domain of that URL was the LMS base URL.

Reply:
just wanted to throw this question, can you tell if openedX installations are using edx-enterprise? i think the answer is no, but just wanted to ask.

Reply:
Here's the [devstack setting](https://github.com/edx/edx-platform/blob/master/lms/envs/devstack.py#L350) to enable/disable enterprise integration (use of edx-enterprise). Enterprise integration is currently enabled by default in devstack.